deps.list

```diff
@@ -3,8 +3,7 @@ GO_VERSION=1.19.3
 GOOGLE_API_VERSION=9de27c9c549173eac194ec80153963234a05a8b5
 GRPC_GATEWAY_VERSION=2.14.0
 GRPC_RUST_VERSION=0.8.3
-GRPC_SWIFT_VERSION=1.13.0
```

Comment:
Why is this removed?

Reply:
My bad, made a mistake resolving conflicts :see_no_evil: